includes/wc-update-functions.php

```diff
@@ -2046,3 +2046,19 @@ function wc_update_370_mro_std_currency() {
 function wc_update_370_db_version() {
 	WC_Install::update_db_version( '3.7.0' );
 }
+
+/**
+ * Increase column size
+ */
+function wc_update_3100_increase_size_of_column() {
+	global $wpdb;
+	$wpdb->query( "ALTER TABLE {$wpdb->prefix}wc_product_meta_lookup MODIFY COLUMN `min_price` decimal(19,4) NULL default NULL" );
+}
```

Comment:
I think the `max_price` column should be updated as well here, no?

Reply:
Yes. I have updated the code.